pkg/proxy/ipvs/proxier_test.go

```diff
@@ -6089,3 +6100,44 @@ func TestNoEndpointsMetric(t *testing.T) {
 		}
 	}
 }
+
+func TestKubeNodePortExceptLoopBackIP(t *testing.T) {
```

Comment:
Nit: rename this function

Reply:
sorry, forget to rename this function, done, PTAL